fix: ensure `elementTemplates.createElement` supports latest features

marstamm opened the pull request:
- `zeebe:property` bindings (cf. https://github.com/bpmn-io/bpmn-js-properties-panel/issues/731)
- optional bindings (cf. https://github.com/bpmn-io/bpmn-js-properties-panel/issues/761)

<!--

Thanks for creating this pull request!

Please make sure to link the issue you are closing as "Closes #issueNr". 
This helps us to understand the context of this PR.

-->

philippfromme commented:
I'm confused. How did I miss this? What exactly breaks or isn't working without this? Is there a related GitHub issue?

marstamm commented:
> I'm confused. How did I miss this? What exactly breaks or isn't working without this? Is there a related GitHub issue?

I assume this is related to the `zeebe:property` bindings? I just found this while working on the optional binding topic and fixed it right away. I did not create an issue for it.

### How to reproduce it:
- Have a template with `zeebe:PropertyType` (e.g. in the `cloud-templates/complex.json`)
- run
```javascript
// using complex.json
const zeebePropertiesTemplate = elementTemplates.getLatest()[11]
elementTemplates.createElement(zeebePropertiesTemplate)

// Throws
// TemplateElementFactory.js?fc1d:98 Uncaught TypeError: Cannot read properties of undefined (reading 'create')
//     at eval (TemplateElementFactory.js?fc1d:98:23)
```
### Expected behavior
I can create an element with zeebe:property binding from a template

### root cause 
This happens because there is no provider registered for `zeebe:property` in the TemplateElement factory. I added this factory in [caa134b](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/762/commits/caa134b495a5e4b7a074f255060689e3b5f68a2f)

marstamm commented:
For optional bindings, we have https://github.com/bpmn-io/bpmn-js-properties-panel/issues/761 as the issue

philippfromme commented:
Okay, thank you for the detailed explanation!
